src/JWT.ts

```diff
@@ -91,6 +91,8 @@ export interface PublicKeyTypes {
   [name: string]: string[]
 }
 export const SUPPORTED_PUBLIC_KEY_TYPES: PublicKeyTypes = {
+  ES256: ['JsonWebKey2020', 'EcdsaSecp256r1VerificationKey2022'],
+  'ES256-R': ['JsonWebKey2020', 'EcdsaSecp256r1VerificationKey2022'],
```

Comment:
Is `ES256-R` really needed? Afaik no one uses this?

Reply:
It clearly seemed like a non-standard key type. Adding recoverySigner , was a "oh why not" kind of thing. But 'ES256-R' and its earlier 'ES256-K' were use at your own discretion kind of things.

Reply:
reference: https://github.com/decentralized-identity/did-jwt/issues/146

Reply:
I agree with @oed, `ES256-R` is not needed.
`ES256K-R` existed as a pattern and was in use before it got a name.
It exists now to allow for `blockchainAccountId` verification methods to be used as verifiers.
We could probably phase it out of existence if we accept 1 bit less of security for ES256K + blockchainAccountId

Reply:
Okay, I have the code for ES256-R. I can archive it for now, and reconsider it later. (?) The reason for this PR was to support something like https://github.com/bshambaugh/key-did-provider-p256/blob/master/src/index.ts . This is a non-functional sketch (at the moment) to support EIP-2844 (https://eips.ethereum.org/EIPS/eip-2844) . At the EthDenver Hackathon I heard about https://f-o-a-m.github.io/foam.developer/foamlite/end-node.html which mentions use of a recovery method that grabs the public key based on a signature. (I discovered that they are also using LoRa.) I literally just read this 5 or 10 minutes ago so I am not educated in the use nor in foam. I think that their protocol is even lower level than what I am attempting.

https://www.youtube.com/watch?v=Z8Wf7Srsg5U&t=50m20s is a Ceramic Community Call that mentions Joel's (oed's) introduction of EIP-2844 to my problem which sent me down this road. Since this (described in the Call) is a preexisting project I may not apply it (or may not be allowed to) to the EthDenver hackathon. In the next week or so, I may in fact try something else. For context, this is the project I was mentioning in the Community Call: https://github.com/bshambaugh/BlinkyProject .

Reply:
> I agree with @oed, `ES256-R` is not needed. `ES256K-R` existed as a pattern and was in use before it got a name. It exists now to allow for `blockchainAccountId` verification methods to be used as verifiers. We could probably phase it out of existence if we accept 1 bit less of security for ES256K + blockchainAccountId

I'll have to peruse the code to understand the context. Since Joel knows about what I have been up to, I assume I can just prune it out for now.